js/apps/admin-ui/src/components/dynamic/url-component.css

```diff
@@ -0,0 +1,3 @@
+.keycloak__identity-providers__url_component > .pf-c-form__group-control {
```

Comment:
This type of selector is sure to break over time as we do upgrades, as it is too specific. If you need to add margins or padding to certain elements I recommend using the [utility classes from PatternFly](https://v4-archive.patternfly.org/v4/utilities/spacing).

Reply:
I was asked in previous PR reviews to specialize the CSS class of the new component in place of using the existing selector `.keycloak__identity-providers__saml_link > .pf-c-form__group-control` from standard `saml-general-settings.css` https://github.com/keycloak/keycloak/blob/a7e5c861cc0623f39804b0b810c218d2febb0492/js/apps/admin-ui/src/identity-providers/add/saml-general-settings.css#L2

therefore I created `.keycloak__identity-providers__url_component > .pf-c-form__group-control` just for this. I don't know if there's a better way but I tried to stick to what's already available (I honestly wouldn't know how to do differently!)

Reply:
Perhaps you could apply this class to the `FormattedLink` component?

Reply:
Per my understanding (not a webdesign/CSS expert, _at all_) it has to be added to the parent `FormGroup` in the same way it's applied to the parent `FormGroup` in https://github.com/keycloak/keycloak/blob/c86620963d3349daa7323e0b41501f8fe649fd2c/js/apps/admin-ui/src/identity-providers/add/SamlGeneralSettings.tsx#L49

Therefore not to `FormattedLink`